### PR TITLE
Task 32

### PR DESCRIPTION
**Pull request by j-ae**
[Task 32](https://github.com/Appers-lab/billihomes-frontend/issues/50)
@pmvald, please review.

### CODE REVIEW

**Review by pmvald — CHANGES_REQUESTED** [review]
@j-ae This is cool. I'm happy with the design. The only thing is that we should use ::part to apply styles directly to the label instead of applying style to the li element. I made code reviews regarding this. Otehr than that it is good.

File: -sass/styles/_lists.m.scss [hunk 1]
```diff
@@ -73,3 +73,36 @@
...
+    &::after {
+      content: "";
+      display: block;
+      margin-left: 2*$spacer;
+      border-bottom: 1px solid #{$light-grey};
+      padding-bottom: 0.8*$spacer;
+    }
+  }
+
+  &--no-item-border {
```
[the remark this answers is not in the document]
Comment (j-ae) [reply -sass/styles/_lists.m.scss]: ".s-checkox-list" isn't a custom-element. so .s-checkox-list::part(label) won't work. right?

**Comment by j-ae** [comment]
I applied the style of the bottom line to <label> using ::part. (it was applied to <li> before.)

**Review by pmvald — APPROVED** [review]
@j-ae cool!

**Pull request merged**